lib/watir-webdriver/browser.rb

```diff
@@ -10,6 +10,7 @@ class Browser
     include Waitable
 
     attr_reader :driver
+    attr_reader :checkers
```

Comment:
Try to think from a user/developer perspective who is not familiar with Watir and it's concepts yet - would it be easier to understand if they'd read word `Checkers` or `AfterHooks`. We both are not target audience for this change and i might be wrong, but it seems to me that `AfterHooks` is more self-explanatory.